### DISCUSSION TITLE
Historic Data

### DISCUSSION THREAD

**Discussion by whitead** [weather-gov/api#470] [Q&A] [answered]
Hi, I'm trying to get historic forecasts in this format. Any advice of how to do that, including resources beyond this API? Thanks!

**Comment by JJKraw** [comment, marked as answer]
I'm pretty sure this API does not keep forecast history. You'd have to fetch and save them on your own.

**Reply by scadergit** [reply]
Effectively, correct. It won't be the same API, but you can look at climate data: https://www.weather.gov/wrh/climate.

**Reply by jah1007** [reply]
Technically speaking climate data is past observational data not past forecast data.

**Reply by whitead** [reply]
Yes, I would like past forecast data for some ML work specifically, not observational data. I guess I'll have to store myself

**Reply by scadergit** [reply]
> Technically speaking climate data is past observational data not past forecast data.

Correct, and the API is more about active weather not performance metrics.

**Reply by Shom770** [reply]
> Effectively, correct. It won't be the same API, but you can look at climate data: https://www.weather.gov/wrh/climate.

How would you get info from this? It seems like a dynamic website for getting information

**Comment by StephenClouse** [comment]
For historical forecast data the primary source is going to be the [NCEI NDFD archive](https://www.ncei.noaa.gov/products/weather-climate-models/national-digital-forecast-database).

Unfortunately they don't offer the data in a JSON format like API (NDFD deals in [GRIB](https://en.wikipedia.org/wiki/GRIB) format) but NDFD and our group do share the same source data.
